lib/providers/json-webhook-provider.ts

```diff
@@ -20,13 +21,22 @@ export function findEndpointsMatchingFilter(filter: OrderFilter, definition: Web
   const catchallEndpoints = definition['*'] ?? []
   endpoints.push(...catchallEndpoints)
 
-  const filterKeys = Object.keys(filter) as FILTER_FIELD[]
-  const filterMapping = definition.filter
-  for (const filterKey of filterKeys) {
-    const filterValue = filter[filterKey]
-    if (filterValue && Object.keys(filterMapping[filterKey]).includes(filterValue)) {
-      const filterEndpoints = filterMapping[filterKey][filterValue]
-      endpoints.push(...filterEndpoints)
+  /**
+   * ignore limit orders when notifying quoters
+   */
```

Comment:
This is a jsdoc-comment (with the `/**`), and should only be used to document functions/classes/variables - things that may be exported. For an implementation detail comment, use `/*` or just an inline `//`. This should be linted - it may be worth configuring the linter for this for the future.

this is also strange because it speaks directly to quoters, but quoters are abstracted out of this function.

```suggestion
  // do not notify quoters of limit orders
```